UI/UnoZIndex/UnoZIndex/UnoZIndex/UnoZIndex/Presentation/MainPage.xaml

```diff
@@ -18,144 +31,137 @@
 		</Grid.RowDefinitions>
 		<utu:NavigationBar Content="Tap on invidual cards to bring to the front" />
```

Comment:
We should just use a TextBlock or something here, the NavBar will look weird with a long title on mobile

Reply:
![image](https://github.com/unoplatform/Uno.Samples/assets/4793020/89d9265f-aacf-4ff9-bfbb-960342b5a60d)